### DISCUSSION TITLE
How/where might I go about adjusting the colors of the frontend?

### DISCUSSION THREAD

**Discussion by MostHated** [Koenkk/zigbee2mqtt#6404] [General]
Hey there,
    I tried to look around a bit but didn't see much mentioned about the frontend other than the config on the docs page. I was wondering where might I need to look if I wanted to change the colors around on the pages/tables of the frontend? I am using Home Assistant (though, not the Hassio OS) and running it through the HA integration. I looked through the files that were included with that but didn't see anything really pertaining to the actual application UI, so I figure it must be within the app itself. 

I am guessing that since it's run in docker, I need to expose the files on disk and edit them there? If anyone has done this and can at least point me to a good starting place, it would be greatly appreciated.

Thanks,
-MH

**Comment by timoline** [comment]
https://github.com/nurikk/z2m-frontend

Here is the source of the frontend, 
But you cannot change some colors that easy
